src/classes/ExpenseController.cls

```diff
@@ -0,0 +1,38 @@
+public class ExpenseController {
+    
+    private String queryString = '';
+    
+    
+    public List<Expense__c> getExpenses(){
+        
+        List<Expense__c> results = Database.query('SELECT Category__c, RecordType.Name, Amount__c, Tax__c, Client_Company__c, Submitted__c, '+
```

Comment:
In salesforce, there's a governor limit over SOQL queries. You can't query more than 50K records, so using unbounded queries is not a good idea. Please add limits to all your queries.